modules/installation-images-samples-disconnected-mirroring-assist.adoc

```diff
@@ -11,7 +11,11 @@ During installation, {product-title} creates a config map named `imagestreamtag-
 
 The format of the key for each entry in the data field in the config map is `<image_stream_name>_<image_stream_tag_name>`.
 
-During a disconnected installation of {product-title}, the status of the Cluster Samples Operator is set to `Removed`. If you choose to change it to `Managed`, it installs samples. 
+During a disconnected installation of {product-title}, the status of the Cluster Samples Operator is set to `Removed`. If you choose to change it to `Managed`, it installs samples.
+[NOTE]
+====
+The use of clusters in a restricted or discontinued network may require actions in services external to your network. Some example services include: Github, RubyGems, PyPi, etc. There may be necessary processes to take that allow the cluster samples operators's objects obtain the services they require or alternative actions.
```

Comment:
include GitHub, Github, RubyGems, PyPi, and others. 

(we try to avoid . etc. usage in the docs)

There might be some additional steps that you must take to allow Cluster Sample Operator objects to obtain the services they require.

(not sure what is meant by alternative actions here)